aws_batch/pipeline.py

```diff
@@ -301,6 +302,29 @@ def generate_json_from_taxid_counts(sample, rawReadsInputPath, taxidCountsInputP
     with open(jsonOutputPath, 'wb') as outf:
         json.dump(output_dict, outf)
 
+def combine_json(inputPath1, inputPath2, outputPath):
+    with open(inputPath1) as inf1:
+        input1 = json.load(inf1).get("pipeline_output")
+    with open(inputPath2) as inf2:
+        input2 = json.load(inf2).get("pipeline_output")
+    total_reads = max(input1.get("total_reads"),
+                      input2.get("total_reads"))
+    remaining_reads = max(input1.get("remaining_reads"),
```

Comment:
Is this the right accounting I wonder?

Reply:
Total_reads is the same for both files, remaining_reads can differ. 
Really, remaining_reads kind of becomes meaningless: there should be remaining_reads_nt and remaining_reads_nr.
We're planning to change the statistics on the pipeline run and add more of them, so I think I'll keep this preliminary solution until we refactor the whole thing.